src/test/deploy.test.ts

```diff
@@ -34,39 +34,40 @@ test('finbot > starts a deploy', (t) => {
 
   respondStub.callsArgWith(1, response);
 
-  const expectedLink = `https://ci.appveyor.com/project/${account}/${project}/deployment/${deploymentId}`
+  const expectedLink = `https://ci.appveyor.com/project/${account}/${project}/deployment/${deploymentId}`;
 
-  const appVeyor = MockAppVeyor.deploys({
-    link: expectedLink
-  });
+  const deployResponse = { link: expectedLink };
+  const deployPromise = Promise.resolve(deployResponse);

```

Comment:
Also this way the control flow is guaranteed, while still using a promise, so you don't need to guess that it will be done before `process.nextTick` (which it won't anyway).